pyproject.toml

```diff
@@ -260,4 +275,8 @@ spaces_indent_inline_array = 4
 trailing_comma_inline_array = true
 overrides."project.classifiers".inline_arrays = false
 overrides."tool.coverage.paths.source".inline_arrays = false
+overrides."tool.hatch".first = [
+    "build",
+    "version",
+]
```

Comment:
This is frustrating, something to do with [TOML array of tables](https://toml.io/en/v1.0.0#array-of-tables). Same thing discussed in https://github.com/pappasam/toml-sort/issues/94.